belt-block/src/consts.rs

```diff
@@ -0,0 +1,179 @@
+///```
+/// for r in (5..32).step_by(8) {
+/// println!("pub const H{}: [u32; 256] = [", r);
+/// for x in 0..256 {
+///    let mut y: u32 = H[x] as u32;
+///    y = y << r | y >> (32 - r);
```

Comment:
If this code tests correctness of the expanded tables, it's worth to do it as tests. You can find a similar [code](https://github.com/RustCrypto/block-ciphers/blob/master/magma/src/sboxes.rs#L554-L576) in the `magma` crate.

Reply:
Added test for extended blocks